org.eclipse.jdt.text.tests/src/org/eclipse/jdt/text/tests/codemining/ParameterNamesCodeMiningTest.java

```diff
@@ -124,6 +124,47 @@ public class Foo {
 		assertEquals(3, fParameterNameCodeMiningProvider.provideCodeMinings(viewer, new NullProgressMonitor()).get().size());
 	}
 
+	@Test
+	public void testRecordClasses() throws Exception {
+		String contents= """
+			public class Ant {
+				record Ca (int size){
+
+				}
+
+				Ca c = new Ca(0);
+
+				}
+			""";
+		ICompilationUnit compilationUnit= fPackage.createCompilationUnit("Ant.java", contents, true, new NullProgressMonitor());
+		JavaEditor editor= (JavaEditor) EditorUtility.openInEditor(compilationUnit);
+		fParameterNameCodeMiningProvider.setContext(editor);
+		JavaSourceViewer viewer= (JavaSourceViewer)editor.getViewer();
+		waitReconciled(viewer);
+
+		assertEquals(0, fParameterNameCodeMiningProvider.provideCodeMinings(viewer, new NullProgressMonitor()).get().size());
+	}
+
+	@Test
+	public void testRecordClasses2() throws Exception {
+		String contents= """
+				public class Ant {
+					record Ca (int size, int color){
+
+					}
+
+					Ca c = new Ca(0,0);
+
+				}
+			""";
+		ICompilationUnit compilationUnit= fPackage.createCompilationUnit("Ant.java", contents, true, new NullProgressMonitor());
+		JavaEditor editor= (JavaEditor) EditorUtility.openInEditor(compilationUnit);
+		fParameterNameCodeMiningProvider.setContext(editor);
+		JavaSourceViewer viewer= (JavaSourceViewer)editor.getViewer();
+		waitReconciled(viewer);
+		assertEquals(2, fParameterNameCodeMiningProvider.provideCodeMinings(viewer,  new NullProgressMonitor()).get().size());
+	}
+
```

Comment:
Remove this test, it does the same as `testRecordConstructorOK`: it tests records with >1 arg:

```suggestion
```

Reply:
done